docs/services/collaboration-architect.md

```diff
@@ -0,0 +1,65 @@
+# Collaboration Architect
+
+Collaboration is at the heart of what makes our hackweeks so enlivening and successful. The role of the Collaboration Architect is to identify and remove barriers that inhibit the participants' ability to voice and contribute to their hackweek experience.  The Collaboration Architect develops strategies, processes, and structured conversations that amplify inclusion, shared understanding, brainstorming, co-mentoring, and collaboration.
+
```

Comment:
Style comment: The paragraph uses a mix and match of one and two spaces after a comma.